### ISSUE TITLE
[BUG] Classname 'max-h-24' incorrectly flagged as invalid

### ISSUE THREAD

**Issue by chadly** [francoismassart/eslint-plugin-tailwindcss#12]
Linting with `tailwindcss/no-custom-classname` rule and this markup:

```js
<div className="max-h-24" />
```

yields

```
Classname 'max-h-24' is not a Tailwind CSS class!
```

but [it is a valid class](https://tailwindcss.com/docs/max-height).

**Comment by francoismassart** [comment]
Hi @chadly,
Thank you for sharing your bug report, I fixed the issue on v1.5.1.
It was just published on npmjs few minutes ago:
https://www.npmjs.com/package/eslint-plugin-tailwindcss

Let me know if you encounter any other bugs, feedback is always useful 👍
